dwa_local_planner/include/dwa_local_planner/dwa_planner_ros.h

```diff
@@ -136,10 +136,11 @@ namespace dwa_local_planner {
 
       /**
        * @brief  Check if the goal pose has been achieved.
-       * The arguments are required by MBF controller API, but are ignored.
+       * @param dist_tolerance The distance tolerance in which the current pose will be partly accepted as reached goal
+       * @param angle_tolerance The angle tolerance in which the current pose will be partly accepted as reached goal
        * @return True if achieved, false otherwise
        */
-      bool isGoalReached(double, double);
+      bool isGoalReached(double dist_tolerance, double angle_tolerance);
```

Comment:
S: Since you're changing this function, maybe add `override` keyword to it :shrug:

Reply:
ok. It should be added to all the virtual methods of `mbf_costmap_core::CostmapController` then...?

Reply:
To make it "correct", yes. But I only suggested this one since you're already changing this line anyway :shrug:

Reply:
I would not waste much brain on kipping DWA pretty and tidy,,, My bet is that we will move to something else soon, either TEB or DWB

Reply:
done